pkg/ddevapp/drupal/backdrop/settings.ddev.php

```diff
@@ -19,6 +19,4 @@
 
 $database = "$driver://{{ $config.DatabaseUsername }}:{{ $config.DatabasePassword }}@$host:$port/{{ $config.DatabaseName }}";
```

Comment:
I guess this was a previous bug? But the `$database = "$driver://{{ $config.DatabaseUsername }}:{{ $config.DatabasePassword }}@$host:$port/{{ $config.DatabaseName }}";` should be inside the 
```
if (empty(getenv('DDEV_PHP_VERSION') && getenv('IS_DDEV_PROJECT') == "true")) {
```
block, shouldn't it?

In that situation, the *only* think DDEV touches in Backdrop is the hash salt, making much of this conversation irrelevant. We don't touch anything except hash_salt unless we're running inside DDEV.